app/com/linkedin/drelephant/schedulers/AirflowScheduler.java

```diff
@@ -58,10 +62,14 @@ public AirflowScheduler(String appId, Properties properties, SchedulerConfigurat
   }
 
   private void loadInfo(String appId, Properties properties) {
-    // Update the 4 Ids
+    // examples:
+    // my_amazing_task_id
     _taskId = properties.getProperty(AIRFLOW_TASK_ID);
+    // 2016-06-27T01:30:00
```

Comment:
On second thought I'm going to modify `getJobExecId` to return `${_dagId}/${_dagRunExecutionDate}/${_taskId}/${_taskInstanceExecutionDate}` since technically two different task instances could exist for the dame dag and run at the same time but be part of different dag runs.